common/src/main/java/com/wynntils/gui/screens/WynntilsQuestBookScreen.java

```diff
@@ -379,6 +388,20 @@ private void renderTooltip(PoseStack poseStack, int mouseX, int mouseY) {
             }
         }
 
+        if (this.hovered instanceof SortOrderWidget) {
+            switch (questSortOrder) {
+                case LEVEL -> tooltipLines = List.of(
+                        new TranslatableComponent("screens.wynntils.wynntilsQuestBook.sort.level.name"),
+                        new TranslatableComponent("screens.wynntils.wynntilsQuestBook.sort.level.description"));
+                case DISTANCE -> tooltipLines = List.of(
+                        new TranslatableComponent("screens.wynntils.wynntilsQuestBook.sort.distance.name"),
+                        new TranslatableComponent("screens.wynntils.wynntilsQuestBook.sort.distance.description"));
+                case ALPHABETIC -> tooltipLines = List.of(
+                        new TranslatableComponent("screens.wynntils.wynntilsQuestBook.sort.alphabetical.name"),
+                        new TranslatableComponent("screens.wynntils.wynntilsQuestBook.sort.alphabetical.description"));
+            }
+        }
```

Comment:
Maybe could be stored in the enum itself?

Reply:
Same as texture